modules/ROOT/pages/anypoint-insights-agentic.adoc

```diff
@@ -0,0 +1,63 @@
+= Monitoring Agentic Assets
+
+//TODO: VERIFY API MANAGER
+In US and EU Cloud instances of Anypoint Platform, navigate from Agent Visualizer or API Manager to view detailed metrics on agentic assets (agent, broker, or MCP server instances).
+
+* Agent: An autonomous software component that uses goals, context, and available tools (often via an LLM) to decide and execute actions on behalf of a user or system.
+* Broker: An intelligent routing service that coordinates task delegation across specialized agents in your enterprise. Brokers originate from MuleSoft and are referenced in the agent network YAML. 
+* MCP server: A service that implements the Model Context Protocol to expose tools and data to AI clients, enabling LLMs to invoke external capabilities through a standard interface.
+
+For information about Agent Visualizer, see xref:agent-visualizer::index.adoc[]. 
+
+//TODO: PREREQS: sub tier, permissions, instances on API Manager or Agent Visualizer? anything else?
+
+//TODO: LOOKBACK settings 24 hours, anything else?
+//TODO: AUTO-REFRESH INTERVALS 
```

Comment:
@sathishpaulleo For the auto-refresh interval options, can you let me know what we allow? Mock doesn't say.

```suggestion
//TODO: AUTO-REFRESH INTERVALS 
```